liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/applications/DeployMojoSupport.java

```diff
@@ -108,6 +108,29 @@ protected void installLooseConfigWar(MavenProject proj, LooseConfigData config)
                     MessageFormat.format(messages.getString("error.project.not.compile"), proj.getId()));
         }
 
+        if (proj.getProperties().containsKey("container") ||
+            (System.getProperty("container") != null)) {
```

Comment:
Why does it have to check both System and project properties here? And if there is a reason, then shouldn't every place that needs to check one have to check the other one too?

Reply:
Removed the need for System.getProperty("container")